.github/workflows/xerox.yml

```diff
@@ -0,0 +1,26 @@
+name: put workflows everywhere
+
+on:
+  push:
+    branches: [ master ] #or main
+  workflow_dispatch:
+  
+
+jobs:
+
+  replicate_changes:
+
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v2
+        - name: Replicating global workflow
+          uses: derberg/global-workflows-support@v0.0.2
+          with:
+            github_token: ${{ secrets.COPYTOKEN }}
```

Comment:
how does this know to look under your account @paulijones?

Reply:
it doesn't, it's pulling the secret from this specific repos [secrets](https://github.com/transposit-connectors/workflows/settings/secrets/actions) 
i just generated the access token as me and stuck it there. We can certainly replace with with one with the same scopes from a generic account if we have that